Makefile

```diff
@@ -23,7 +23,11 @@ test: test-unit
 
 .PHONY: test-unit
 test-unit: clean
-	mvn test
+	mvn test -Dgroups="Unit"
+
+.PHONY: test-integration
+test-integration: clean
```

Comment:
There'll be a separate pipeline PR to call this new target from the Concourse build.